### PR TITLE
Guiga004 scope download classification robust to server errors - merge fix

### PR DESCRIPTION
**Pull request by guiga004**

### CODE REVIEW

File: scope/fritz.py [hunk 1]
```diff
@@ -90,7 +91,19 @@ def api(
         kwargs["json"] = data
     elif method == "GET":
         kwargs["params"] = data
-    response = requests.request(**kwargs)
+
+    for attempt in range(max_attempts):
```
Comment (mcoughlin) [inline scope/fritz.py:95]: There should be some kind of sleep here?

Comment (mcoughlin) [reply scope/fritz.py:95]: Probably that sleep should be set as a function argument. Defaulting to 5 seconds?

**Review by mcoughlin — APPROVED** [review]
Just need to add some kind of sleep in your retry.

**Review by bfhealy — APPROVED** [review]
I agree with Michael's comment, otherwise looks good!

**Pull request merged**